src/controllers/auth.controller.js

```diff
@@ -2,22 +2,18 @@ import { generateToken } from '../helpers/auth';
 import response from '../helpers/response';
 import { User } from '../database/models';
 
-/**
- * @class AuthController
- */
-export default class AuthController {
   /**
    * Creates a user successfully
    *
```

Comment:
The response obj has been changed. Make the appropriate adjustment. a response should now look like return `response(res, 400, "message", {data});`